src/main/resources/assets/iris/lang/ko_kr.json

```diff
@@ -4,17 +4,19 @@
   "iris.keybind.reload": "쉐이더 새로고침",
   "iris.keybind.shaderPackSelection": "쉐이더팩 선택 창",
   "iris.keybind.toggleShaders": "쉐이더 ON/OFF 전환",
-  "iris.keybinds": "Iris (아이리스)",
+  "iris.keybinds": "Iris",
   "iris.shaders.reloaded.failure": "%s 사유로 인하여 쉐이더를 다시 불러오지 못했습니다!",
   "iris.shaders.toggled.failure": "%s 사유로 인하여 쉐이더를 켜지/끄지 못했습니다!",
   "iris.sodium.failure.title": "아이리스를 불러오지 못했습니다!",
-  "iris.sodium.failure.download": "Sodium(소듐) 다운로드",
-  "iris.sodium.failure.reason.notFound": "아이리스를 최고의 성능으로 쓰기 위해선 Sodium(소듐)이 필요하지만, 소듐이 설치되어 있지 않습니다. 소듐을 내려받아 mods 폴더에 넣어 게임을 다시 실행해주세요.",
-  "iris.sodium.failure.reason.incompatible": "아이리스를 최고의 성능으로 쓰기 위해선 Sodium(소듐)이 필요하지만, 호환되지 않는 버전의 소듐을 사용하고 있습니다. 아이리스와 맞는 버전의 소듐을 내려받아 mods 폴더에 넣고, 호환되지 않는 기존 버전의 소듐을 제거한 후 게임을 다시 실행해주세요.",
+  "iris.sodium.failure.download": "Sodium 다운로드",
+  "iris.sodium.failure.reason.notFound": "Sodium이 설치되어 있지 않습니다. Sodium을 내려받아 mods 폴더에 넣어 게임을 다시 실행해주세요.",
+  "iris.sodium.failure.reason.incompatible": "호환안되는 버전의 Sodium을 사용하고 있습니다. 아이리스와 맞는 버전의 Sodium을 내려받아 mods 폴더에 넣고, 호환 안되는 기존 버전의 소듐을 제거 후 게임을 다시 실행해주세요.",
```

Comment:
These changes appear to be removing the note about how Iris requires Sodium for good performance. I feel like this is an important detail of the message since it justifies *why* the user needs Sodium, so I'd appreciate if it were retained in the translated version of the message.